src/main/java/org/elasticsearch/index/query/CommonTermsQueryBuilder.java

```diff
@@ -171,14 +239,18 @@ public CommonTermsQueryBuilder queryName(String queryName) {
         return this;
     }
 
+    public String queryName() {
+        return this.queryName;
+    }
+
     @Override
     public void doXContent(XContentBuilder builder, Params params) throws IOException {
         builder.startObject(NAME);
-        builder.startObject(name);
+        builder.startObject(fieldName);
 
         builder.field("query", text);
-        if (disableCoords != null) {
-            builder.field("disable_coords", disableCoords);
+        if (disableCoord != null) {
+            builder.field("disable_coord", disableCoord);
```

Comment:
oh boy this never worked through java api it seems, because disable_coords should have been disable_coord? if this is the case this same fix should be ported to master and probably 1.x too.